api/notd/metadata_manager.py

```diff
@@ -0,0 +1,38 @@
+import json
+import logging
+import os
+import urllib.request
+
+from web3 import Web3
+
+from notd.model import TokenMetadata
+
+_EMPTY_STRING = '_EMPTY_STRING'
+w3 = Web3(Web3.HTTPProvider(f'https://mainnet.infura.io/v3/{os.environ["INFURA_PROJECT_ID"]}'))
+
+def retrieve_token_metadata(registryAddress: str,tokenId:int):
+    with open('./contracts/IERC721Metadata.json') as contractJsonFile:
```

Comment:
when this is in a class this would be better in the __init__ so you dont have to do it every time